### ISSUE TITLE
Multi monitor support

### ISSUE THREAD

**Issue by natcl** [bencevans/screenshot-desktop#3]
On Windows on a multi monitor setup I only get the main monitor, would there be a way to get all screens ?

**Comment by bgschiller** [comment]
I'd like to try and tackle this.

It seems like we'll need to have a function to list devices, so that a user can choose which screen[s] to capture. On mac, `system_profiler SPDisplaysDataType` will present a list of which screens are available, and we can capture multiple with https://stackoverflow.com/a/21673829/1586229

What should the Promise resolve to if there are multiple screens? An array of buffers? Or maybe the user must pick exactly one screen to snap? That seems to be the simplest choice to me. It is backwards compatible, and means that the display choice will be an optional parameter.

 I'd love any direction or advice for how to do this on windows and linux, as those aren't my primary development platforms.

**Comment by bencevans** [comment]
Hi @bgschiller that'd be grand. I'm seeing this as something along the lines of the following... thoughts?


Screenshot default/first screen:

```js
await screenshot();
```

Screenshot specified screen (anyone know what's used as a screen identifier across platforms?):

```js
await screenshot({screen: identifier});
```

List displays:

```js
await screenshot.listDisplays();
```

Screenshot all displays:

```js
await screenshot.all();
// => [Buffer, Buffer]
```

**Comment by bencevans** [comment]
Added in #7. Released in `v1.3.0`.
